lib/serial_terminal.pm

```diff
@@ -118,6 +118,7 @@ sub download_file {
     $opts{chunk_size}  //= 1024 * 2;
     $opts{chunk_retry} //= 16;
     $opts{force}       //= 0;
+    $opts{timeout}     //= scale_timeout(90);
```

Comment:
this will add another flexibility dimension - to increase/decrease waiting time you need to just adjust Test Suite variable ( yeah in this case I vote for using Test SUite variables :P )